Sources/SotoCore/AWSClient.swift

```diff
@@ -521,7 +521,7 @@ extension AWSClient {
         url: URL,
         httpMethod: HTTPMethod,
         headers: HTTPHeaders = HTTPHeaders(),
-        body: AWSPayload,
+        body: HTTPBody,
```

Comment:
Does the DocC comment need updating?

Reply:
No payload/body are basically same thing. Did you see my slack comment about renaming HTTPBody to AWSPayload?